src/main/kotlin/anttracker/Main.kt

```diff
@@ -5,39 +5,38 @@ Rev 1 - 2024/07/01 Original by Micah
 entry-point and main-menu of AntTracker.
  */
 
-package anttracker
```

Comment:
Can you please clarify why main should not be in `anttracker`?

Reply:
I don't have a reason for this - I am not entirely sure how Kotlin's package management works.

Reply:
The `package` keyword states what directory the file is located in.